app/src/main/java/com/isc/hermes/database/AccountInfoManager.java

```diff
@@ -105,6 +106,28 @@ public String getIdByEmail(String email) throws ExecutionException, InterruptedE
         return jsonObject.getString("_id");
     }
 
+    /**
```

Comment:
You should not use an email update because performance will be poor there. Instead you can use method update using id. And you can get the email from the user repository

Reply:
No matter if it is the ID or the EMAIL, the performance will still be mediocre, but it has been fixed and moved to the MainActivity, because when the session starts it does not load the information from the database, but the one that is returned by the google service.

Reply:
It does matter since if you do it with the email a collection or array is created in the json. Since the email is not a unique attribute according to the database but the id is, so if you take it by means of the id it takes you an object but if you take it with the email it returns a json array with an object